Source/Scene/ModelExperimental/FeatureIdPipelineStage.js

```diff
@@ -49,43 +49,52 @@ FeatureIdPipelineStage.process = function (
   if (featureIdTextures.length > 0) {
     processFeatureIdTextures(renderResources, frameState, featureIdTextures);
   } else {
-    var featureIdAttributeIndex = model.featureIdAttributeIndex;
-    var instances = renderResources.runtimeNode.node.instances;
-
-    var featureIdAttributeInfo = getFeatureIdAttributeInfo(
-      featureIdAttributeIndex,
-      primitive,
-      instances
-    );
-
-    var featureIdAttribute = featureIdAttributeInfo.attribute;
-    var featureIdAttributePrefix = featureIdAttributeInfo.prefix;
-
+    var featureIdAttributePrefix;
     var featureIdAttributeSetIndex;
 
-    // Check if the Feature ID attribute references an existing vertex attribute.
-    if (defined(featureIdAttribute.setIndex)) {
-      featureIdAttributeSetIndex = featureIdAttribute.setIndex;
+    // For 3D Tiles 1.0, the FEATURE_ID vertex attribute is present but the Feature ID attribute is not.
+    // The featureMetadata is owned by the Cesium3DTileContent for the legacy formats.
+    var content = model.content;
+    if (defined(content) && defined(content._featureMetadata)) {
```

Comment:
Can this use the getter instead of the private variable?

```suggestion
    if (defined(content) && defined(content.featureMetadata)) {
```

Reply:
Updated to use the getter